### PR TITLE
Fix: APP-1066: SDK Transfers and PluginContext

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description
- Removes `pluginAddress` from `usePluginClient` hook (hence the billion file updates)
- Reverts plugin client creation to use `FromContext`
- Updates transfers data manipulation to conform to new SDK types
- Update explorer temp DAO addresses

Task: [APP-1066](https://aragonassociation.atlassian.net/browse/APP-1066?atlOrigin=eyJpIjoiOTc3ODJjZGVhZGJmNDRmNTkzYjcwOWJhNWY1ZmQ5Y2QiLCJwIjoiaiJ9)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/hooks/useDaoProposal.tsx [hunk 1]
```diff
@@ -39,10 +39,7 @@ export const useDaoProposal = (
   const [isLoading, setIsLoading] = useState(false);
 
   const {client: globalClient} = useClient();
-  const pluginClient = usePluginClient(
-    pluginAddress,
-    pluginType as PluginTypes
-  );
+  const pluginClient = usePluginClient(pluginType as PluginTypes);
```
Comment (WissenIstNacht) [inline packages/web-app/src/hooks/useDaoProposal.tsx]: I don't think the `PluginType` assertion is necessary.

**Review by RakeshUP — APPROVED** [review]

File: packages/web-app/src/hooks/useDaoTransfers.tsx [hunk 2]
```diff
@@ -26,34 +24,35 @@ export const useDaoTransfers = (
...
+        const transfers = await client?.methods.getTransfers({
+          sortBy: TransferSortBy.CREATED_AT,
+          daoAddressOrEns,
+        });
+
         if (transfers) {
-          if (pendingDepositsTxs.length > 0) {
-            for (let i = 0; i < pendingDepositsTxs.length; ) {
-              const tx = pendingDepositsTxs[i];
+          const deposits = transfers.filter(t => t.type === 'Deposit');
```
Comment (WissenIstNacht) [inline packages/web-app/src/hooks/useDaoTransfers.tsx:33]: While you're at it I think you could remove the `if(transfers)` check, too. (`filter` on an empty array should be ok)

**Review by WissenIstNacht — APPROVED** [review]
Rare to have a PR with more deletions than additions. Feels good! :D

File: packages/web-app/src/hooks/useDaoTransfers.tsx (hunk 2 above)
Comment (Fabricevladimir) [reply packages/web-app/src/hooks/useDaoTransfers.tsx:33]: This is more stylistic. If I remove the if, then `transfers`, `deposits` and `deposit` have potential to be be undefined, so I have to use optional chaining all over the place.

**Pull request merged**